### PR TITLE
Making hearder

### PR DESCRIPTION
**Pull request by PriscilliaAmmeux**

### CODE REVIEW

File: frontend/src/components/Carousel/carousel.css [hunk 1]
```diff
@@ -10,8 +10,7 @@ body {
 .swiper {
   width: 80%;
   height: 100%;
-  position: relative;
-}
```
Comment (cartige) [inline frontend/src/components/Carousel/carousel.css:14]: No closing bracket

**Review by cartige — CHANGES_REQUESTED** [review]
carousel.css file line 14 15

**Review by cartige — DISMISSED** [review]

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1 @@
-node_modules
-package-lock.json
\ No newline at end of file
+node_modules
```
Comment (LoicBrassart) [inline .gitignore]: Don't rollback my modifications, thanks :smirk:

File: backend/.gitignore [hunk 3]
```diff
@@ -1,3 +1,2 @@
 node_modules
 .env
```
Comment (LoicBrassart) [inline backend/.gitignore]: Don't rollback my modifications, thanks :smirk:

File: frontend/.gitignore [hunk 4]
```diff
@@ -23,4 +23,3 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
```
Comment (LoicBrassart) [inline frontend/.gitignore:26]: Don't rollback my modifications, thanks :smirk:

File: frontend/src/components/BurgerMenu/BurgerMenu.css [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+  }
+  .nav-list .nav-link :hover {
+    box-shadow: rgba(0, 0, 0, 0.25) 0px 54px 55px,
+      rgba(0, 0, 0, 0.12) 0px -12px 30px, rgba(0, 0, 0, 0.12) 0px 4px 6px,
+      rgba(0, 0, 0, 0.17) 0px 12px 13px, rgba(0, 0, 0, 0.09) 0px -3px 5px;
+  }
+  .menuContainer {
+    display: flex;
+    position: relative;
+    z-index: 1000;
```
Comment (LoicBrassart) [inline frontend/src/components/BurgerMenu/BurgerMenu.css]: Why do you think you need *that* ?

File: frontend/src/components/Carousel/Carousel.jsx [hunk 6]
```diff
@@ -1,21 +1,16 @@
...
 import { Swiper, SwiperSlide } from "swiper/react";
-import { PropTypes } from "prop-types";
-/* eslint-disable import/no-unresolved */
+
+// eslint-disable-next-line import/no-unresolved
 import "swiper/css";
+// eslint-disable-next-line import/no-unresolved
 import "swiper/css/pagination";
+// eslint-disable-next-line import/no-unresolved
 import "swiper/css/navigation";
```
Comment (LoicBrassart) [inline frontend/src/components/Carousel/Carousel.jsx]: Please restore this, it was better written

File: package.json [hunk 7]
```diff
@@ -1,24 +1,26 @@
...
-  "author": "Wild Code School",
-  "license": "ISC",
-  "devDependencies": {
-    "concurrently": "^7.0.0",
-    "husky": "^7.0.0"
-  }
-}
-
-
+{
```
Comment (LoicBrassart) [inline package.json:1]: Why did you add this file ? Most changes seem unintentional to me

File: package.json [hunk 8]
```diff
@@ -1,24 +1,26 @@
...
+  "author": "Wild Code School",
+  "license": "ISC",
+  "devDependencies": {
+    "concurrently": "^7.0.0",
+    "husky": "^7.0.0"
+  },
+  "dependencies": {
+    "react-icons": "^4.6.0",
+    "swiper": "^8.4.4"
+  }
```
Comment (LoicBrassart) [inline package.json]: Nope! These are packages to install in your `frontend` folder

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Most Evils here could be avoided with one piece of advice: "Don't `git add .`"

**Review by CharlyJuteau — DISMISSED** [review]

**Review by LoicBrassart — DISMISSED** [review]
Saw no Evil

**Comment by LoicBrassart** [comment]
Missing SM validation

**Review by CharlyJuteau — APPROVED** [review]

**Review by CharlyJuteau — DISMISSED** [review]

File: frontend/src/components/Carousel/Carousel.jsx [hunk 9]
```diff
@@ -34,69 +39,81 @@ export default function Carousel() {
...
-      <SwiperSlide>
-        <img
-          alt="Athenes"
-          src="https://evasion-online.com/image-photo/athenes/Athene_Grece-1.jpg"
-        />
-        <p className="text-block">Athenes</p>
-      </SwiperSlide>
+      {europeanCities.map((europeanCity) => {
+        return (
+          <div>
```
Comment (LoicBrassart) [inline frontend/src/components/Carousel/Carousel.jsx]: I think that'll leave a red trace in your console, you're missing the `key` props

File: frontend/src/components/Carousel/Carousel.jsx [hunk 10]
```diff
@@ -34,69 +39,81 @@ export default function Carousel() {
...
+              />
+              <p className="CitiesName">{europeanCity.cityName} </p>
+            </SwiperSlide>
+          </div>
+        );
+      })}
+
+      {restaurants.map((restaurant) => {
+        return (
+          <div>
```
Comment (LoicBrassart) [inline frontend/src/components/Carousel/Carousel.jsx]: Same here

File: frontend/src/components/Carousel/Carousel.jsx [hunk 11]
```diff
@@ -34,69 +39,81 @@ export default function Carousel() {
...
+            <SwiperSlide>
+              <p>{restaurant.TypeName} </p>
+            </SwiperSlide>
+          </div>
+        );
+      })}
+
+      {museums.map((museum) => {
+        return (
+          <div>
```
Comment (LoicBrassart) [inline frontend/src/components/Carousel/Carousel.jsx]: Same

File: frontend/src/components/Carousel/Carousel.jsx [hunk 12]
```diff
@@ -34,69 +39,81 @@ export default function Carousel() {
...
+            <SwiperSlide>
+              <p>{museum.TypeName} </p>
+            </SwiperSlide>
+          </div>
+        );
+      })}
+
+      {parks.map((park) => {
```
Comment (LoicBrassart) [inline frontend/src/components/Carousel/Carousel.jsx]: Same

File: frontend/src/components/Carousel/Carousel.jsx [hunk 13]
```diff
@@ -34,69 +39,81 @@ export default function Carousel() {
...
+    PropTypes.shape({
+      TypeName: PropTypes.string,
+    })
+  ),
+  parks: PropTypes.arrayOf(
+    PropTypes.shape({
+      TypeName: PropTypes.string,
+    })
+  ),
+};
```
Comment (LoicBrassart) [inline frontend/src/components/Carousel/Carousel.jsx]: Congrats on working with proptypes ! :clap:

**Review by LoicBrassart — DISMISSED** [review]
Much better!
But you're missing key props in a lot of places, I wouldn't like to see your console... please fix this asap

**Review by cartige — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
Saw no Evil ?
(I hope this PR gets done quickly, I'm under the impressino some files did a lot of yoyo on it this week
